server/src/test/java/org/elasticsearch/repositories/fs/FsRepositoryTests.java

```diff
@@ -201,16 +200,8 @@ public void testSnapshotAndRestore() throws IOException, InterruptedException {
         }
     }
 
-    private void runGeneric(ThreadPool threadPool, Runnable runnable) throws InterruptedException {
-        CountDownLatch latch = new CountDownLatch(1);
-        threadPool.generic().submit(() -> {
-            try {
-                runnable.run();
-            } finally {
-                latch.countDown();
-            }
-        });
-        latch.await();
+    private void runGeneric(ThreadPool threadPool, Runnable runnable) {
```

Comment:
may as well just inline this now it's a one-liner?

Reply:
ah sorry meant to delete this comment - there's no need to fork at all any more I think

Reply:
:facepalm: right! I've also done some renames! Please see https://github.com/elastic/elasticsearch/pull/88723/commits/d942ffcf9e51dd0a2e8ca068cd1c5abd81286483.